client/app/core/navigation/navigation-controller.js

```diff
@@ -86,6 +87,14 @@ export function NavigationController (Text, Navigation, Session, API_BASE, Shopp
       about: about(),
       sites: sites()
     })
+
+    CollectionsApi.query('settings/help_menu/documentation').then((data) => {
+      const documentation = data.help_menu.documentation
+      const matches = documentation.href.match(/http.*(http.*)/)
```

Comment:
What if this ends up being https?  still work?

Reply:
should continue to work fine because .* accounts for any character after http.   I wrote it intentionally so it would accept http:// or https:// .